tests/ut/ir/transforms/test_materialize_dist_tensor_ctx.py

```
@@ -884,160 +805,97 @@ def test_device_call_without_materialized_context_rejects_synthesized_prefix():
+        @pl.function(type=pl.FunctionType.InCore)
+        def stage(
+            self,
+            data: pld.DistributedTensor[[4], pl.FP32],
+            scratch: pl.Out[pl.Tensor[[4], pl.FP32]],
+        ) -> pl.Tensor[[4], pl.FP32]:
+            return scratch
+
+        @pl.function(type=pl.FunctionType.Orchestration)
+        def main(self, data: pld.DistributedTensor[[4], pl.FP32]):
+            with pl.manual_scope():
+                out, tid = pl.submit(self.stage, data)
```

Ruff (0.16.2)

[warning] Unpacked variable `tid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)